protocol/packages/swap/src/testing/mod.rs

```diff
@@ -55,13 +53,44 @@ where
     CurrencyDTO::<G>::from_symbol_testing::<DexSymbols<G>>(symbol).map_err(Into::into)
 }
 
+#[cfg(any(feature = "testing", test))]
```

Comment:
no need to double this since it's identical to the one in `lib.rs` right above `mod testing`